TUTORIAL.md

```diff
@@ -781,12 +781,8 @@ To learn more about PWAs, check out some recent tutorials I wrote:
 * [Build Your First Progressive Web Application with Angular and Spring Boot](http://developer.okta.com/blog/2017/05/09/progressive-web-applications-with-angular-and-spring-boot) 
 * [Tutorial: Develop a Mobile App With Ionic and Spring Boot](http://developer.okta.com/blog/2017/05/17/develop-a-mobile-app-with-ionic-and-spring-boot)
 
-There's also a number of excellent resources by Google, Pluralsight, and Smashing Magazine:
+There's also a number of excellent resources by Google and Smashing Magazine:
 
 * Addy Osmani at Google I/O '17: [Production Progressive Web Apps With JavaScript Frameworks](https://youtu.be/aCMbSyngXB4)
 * Google's [Progressive Web Apps](https://developers.google.com/web/progressive-web-apps/) homepage, [step-by-step code lab](https://codelabs.developers.google.com/codelabs/your-first-pwapp/), and [instructor-led PWA training](https://developers.google.com/web/ilt/pwa/).
-* Pluralsight's [Getting Started with Progressive Web Apps](https://www.pluralsight.com/courses/web-apps-progressive-getting-started)
-* [A Beginner's Guide To Progressive Web Apps](https://www.smashingmagazine.com/2016/08/a-beginners-guide-to-progressive-web-apps/)
-
-// let's kill the last link here, and add your scotch post's url to the list above. i'll work on getting that URL for you
- 
+* [A Beginner's Guide To Progressive Web Apps](https://www.smashingmagazine.com/2016/08/a-beginners-guide-to-progressive-web-apps/) 
```

Comment:
I decided to keep Smashing Magazine's article since it's easier to read than watch a video IMO.